tests/oeedger8r/edl/array.edl

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 
 enclave  {
+    include "wchar.h"
```

Comment:
This seems like a breaking change.
Do user written edl files that use wchart_t now have to explicitly do this include?

Reply:
It is, but it is intentional. wchar_t is not a builtin C type and anyone who would want to use it in normal C has to include it somehow. In the same way that you would need to include a header for a struct defined by libc if you want to use it.

It also isn't really an official contract. People were getting wchar_t for free by chance, it is not documented that wchar_t is a builtin type.

Even still, I do agree this has the potential to break existing code and should be done in a separate change with an addition to the changelog.

Reply:
Just so I understand what you are saying,

edger8r has always had first class support for wchar, as seen in the Lexer,
https://github.com/openenclave/openenclave/blob/261947bce47abcbe195f39f32d595f7061ec18fb/tools/oeedger8r/intel/Lexer.mll#L69-L77

and Parser,
https://github.com/openenclave/openenclave/blob/261947bce47abcbe195f39f32d595f7061ec18fb/tools/oeedger8r/intel/Ast.ml#L49-L53

When you say `people were getting wchar_t for free by chance`, are your referring to edger8r generated code or something else?

Also note that edger8r supports a lot of other types like `uint64_t`, 'size_t` that are similar to `wchar_t` in that they are are not C builtin types, but rather typedefs of C builtin types. Can you clarify why the user would have to include "wchar.h" to access wchar_t, but would not need to include "stdint.h" to access 'uint64_t'?

Reply:
@jhand2 I am concerned about this breaking change as it requires every developer using edl to change what they include. It also deviates from Intel's edger8r. Is there a way around this? Can we make it so devs are not required to #include wchar.h?

Reply:
See comment at bottom of PR